source/include/webpa_adapter.h

```diff
@@ -14,6 +14,7 @@
 #include <stdint.h>
 #include <wdmp-c.h>
 #include <cimplog.h>
+#include <libparodus.h>
```

Comment:
What is this for?

Reply:
Included this "libparodus.h" header in webpa_adapter.c file, to use headers_t struct from wrp

Reply:
yes, agree .

Reply:
can we not use wrp header?